File: PolicyManagement.Api/PolicyManagement.Models/Motor/InspectionFormDataModel.cs
using PolicyManagement.Dtos.Common;
using System;

namespace PolicyManagement.Models.Motor
{
    public class InspectionFormDataModel
    {
        public byte InspectionCompany { get; set; }
        public string InspectionNumber { get; set; }
        public string InspectionDateString { get; set; }
        public DateTime? InspectionDate { get; set; }
        public DateDto InspectionDateDto { get; set; }
        public string InspectionTime { get; set; }
        public string InspectionRemarks { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/motor/inspection-form-data-model.ts
import { IDateDto } from "src/app/app-entites/dtos/common/date-dto";

export interface IInspectionFormDataModel {
    InspectionCompany: number;
    InspectionNumber: string;
    InspectionDateString: string;
    InspectionDateDto: IDateDto|null;
    InspectionTime: string;
    InspectionRemarks: string;
}